src/app/app.component.html

```diff
@@ -1,30 +1,43 @@
-
-
-<main class="main">
-	<div class="wrapper">
-		<div class="container">
-			<h1 style="text-align: center">Congratulations, you made it!</h1>
-			<h3 style="text-align: center">You have offically built your first Carbon Angular app :)</h3>
-
-			<div style="max-width: 800px; margin: 20px 0;">
+<div #container class="container carbon--theme--white">
+	<div class="demo-content" ibmGrid>
+		<div ibmRow>
+			<div ibmCol [columnNumbers]="{'lg':6, 'md':12, 'sm': 12}">
+				<h1 class="title">Congratulations, <br>you made it!</h1>
+			</div>
+			<div ibmCol [columnNumbers]="{'lg':6, 'md':12, 'sm': 12}">
+				<ibm-radio-group style="position: absolute; right: 0">
+					<ibm-radio checked="true" (change)="onWhiteChange()">white</ibm-radio>
+					<ibm-radio (change)="onG10Change()">g10</ibm-radio>
+					<ibm-radio (change)="onG90Change()">g90</ibm-radio>
+					<ibm-radio (change)="onG100Change()">g100</ibm-radio>
+				</ibm-radio-group>
+			</div>
+		</div>
+		<div ibmRow>
+			<p style="margin-top: 2rem">You have offically built your first Carbon Angular app :)</p>
```

Comment:
Lets drop the `:)` personally I don't mind it, but it's out of place in the new design

```suggestion
			<p style="margin-top: 2rem">You have offically built your first Carbon Angular app</p>
```

Reply:
Also that text isn't horizontally aligned with the text above and below it ... seems like a grid/padding issue?